index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+     <link rel="stylesheet" href="styles.css"> 
+</head>
+<body>
+  <header class="tool-bar">
+    <div class="toolbar-dark-compact">
+        <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
+        <nav class="normal-button" >  
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: Once you did a great job changing the requested change by the previous reviewer, I noticed that you didn't match the Logo styling to the one from Figma. I suggest you remove the `underline decoration` with `a { text-decoration: none;  }` and change the `href` attribute value so it can redirect on the home page and not the `w3schools` page i.e.: `<a class="john-doe" href="#">Yasen</a>` and not `<a class="john-doe" href="https://www.w3schools.com">Yasen</a>`